Rebuild-Black-Business/RBB-Website
# Add opacity layer

# Describe your PR
Add opacity layer behind Nav
Prevent scrolling while Nav Slideout is open
Nav height is now fixed to 100px
Set layout to max width of 100 view width

Related to #
Fixes https://trello.com/c/fGyxkki2/138-mobile-navigation-still-needs-opacity-on-the-content

## Pages/Interfaces that will change
Mobile Nav

### Screenshots / video of changes
Before
![image](https://user-images.githubusercontent.com/20157895/84332246-0fdc8680-ab52-11ea-841c-601ef4ff877c.png)

After
![image](https://user-images.githubusercontent.com/20157895/84332181-ecb1d700-ab51-11ea-9836-b4d507b8e394.png)


## Steps to test

1. Open site on Mobile.
2. Click Menu icon
3. -> see opacity layer and that you can't scroll

### Additional notes

There was no change to the logic within the Nav component.
I wrapped the whole Nav in a box with relative positioning which is why you see the changes.
I did change Nav props, making it absolutely positioned (to sit on top of the opacity layer), z-index, and a set height and width. To support the 100px height I did change the vertical padding on higher resolutions but it's centered vertically so that didn't impact anything. I just had to do it because we lowered to 100px height.
